### PR TITLE
More bcrypt stuff

### PR DESCRIPTION
**Pull request by openglfreak**

### CODE REVIEW

File: wine-tkg-git/wine-tkg-patches/hotfixes/hotfixer [hunk 1]
```diff
@@ -963,11 +964,11 @@ fi
 
 # Fixes a regression breaking mostly some game launchers such as Riot client or Minecraft dungeon launcher - https://bugs.winehq.org/show_bug.cgi?id=49990
 # It seems to break riot client on 6.15 while star citizen launcher works fine with and without it, so let's disable on 6.15+
-if ( cd "${srcdir}"/"${_winesrcdir}" && git merge-base --is-ancestor bd27af974a21085cd0dc78b37b715bbcc3cfab69 HEAD ); then
```
Comment (Tk-Glitch) [inline wine-tkg-git/wine-tkg-patches/hotfixes/hotfixer:966]: Was this desired?

Comment (openglfreak) [reply wine-tkg-git/wine-tkg-patches/hotfixes/hotfixer:966]: nope, sorry

Comment (openglfreak) [reply wine-tkg-git/wine-tkg-patches/hotfixes/hotfixer]: fixed

**Comment by Tk-Glitch** [comment]
Thanks openglfreak! :frog: :heart:

**Pull request merged**